frappe/website/doctype/blog_post/blog_post.py

```diff
@@ -101,7 +101,7 @@ def load_comments(self, context):
 				context.comment_text = _('{0} comments').format(len(context.comment_list))
 
 	def set_read_time(self):
-		content = self.content or self.content_html
+		content = (self.content or self.content_html) or ''
```

Comment:
Add the rich text field here not an empty string

Reply:
Okay, my bad.

Reply:
`self.content` is for the Rich Text Content, and the empty string is a fallback if `self.content` is `None`

The error was caused due to None value set in `self.content` (Rich Text box) on empty save